.github/workflows/ghcr.yml

```diff
@@ -0,0 +1,56 @@
+# Mirror DockerHub images used by the Rust project to ghcr.io.
+# Images are available at https://github.com/orgs/rust-lang/packages.
+#
+# In some CI jobs, we pull images from ghcr.io instead of Docker Hub because
+# Docker Hub has a rate limit, while ghcr.io doesn't.
+# Those images are pushed to ghcr.io by this job.
+#
+# Note that authenticating to DockerHub or other registries isn't possible
+# for PR jobs, because forks can't access secrets.
+# That's why we use ghcr.io: it has no rate limit and it doesn't require authentication.
+
+name: GHCR
+
+on:
+  schedule:
```

Comment:
Let's also add a `workflow_dispatch` trigger, so that we can manually trigger the workflow, both to test it, and to make it possible to mirror "on demand", without waiting for midnight :)